stellarphot/settings/models.py

```diff
@@ -558,10 +622,42 @@ class PhotometryOptionalSettings(BaseModelWithTableRep):
     False
     """
 
-    include_dig_noise: bool = True
-    reject_too_close: bool = True
-    reject_background_outliers: bool = True
-    fwhm_by_fit: bool = True
+    include_dig_noise: Annotated[
+        bool,
+        Field(
+            description=(
+                "Should the digitization noise be included in the calculation of "
+                "the noise for each observation?"
+            )
+        ),
+    ] = True
+
+    reject_too_close: Annotated[
+        bool,
+        Field(
+            description="Should sources that are too close be excluded from photometry?"
```

Comment:
I would say "too close together"... but again, that's me. On first read I imagined (OK, so excluding the moon and orbiting satellites...